src/vehicle.cpp

```diff
@@ -4273,7 +4275,7 @@ veh_collision vehicle::part_collision( int part, const tripoint &p,
             }
         }
 
-        coll_velocity = vel1_a * ( smashed ? 100 : 90 );
+        coll_velocity = vel1_a * ( smashed ? 100 : 99 );
```

Comment:
Can be improved further by not scaling here. As mentioned in PR description, I don't know (yet) why it's like this in the first place, so leaving.